centreon/www/front_src/src/Resources/Filter/FilterStorage.cypress.spec.tsx

```diff
@@ -92,32 +92,47 @@ const emptyListData = {
 
 const linuxServersHostGroup = {
   id: 0,
-  name: 'Linux-servers'
+  name: 'Linux-servers',
+  formattedName: 'Linux-servers'
 };
 
 const webAccessServiceGroup = {
   id: 0,
-  name: 'Web-access'
+  name: 'Web-access',
+  formattedName: 'Web-access'
 };
 
+const parameters = [
+  { name: 'search', object_type: null, type: 'text', value: '' },
+  { name: 'sort', value: [defaultSortField, defaultSortOrder] }
+];
+
 const filter = {
   criterias: [
     {
       name: 'resource_types',
-      value: [{ id: 'host', name: labelHost }]
+      type: 'multi_select',
+      value: [{ id: 'host', name: labelHost, formattedName: labelHost }]
     },
     {
       name: 'states',
+      type: 'multi_select',
       value: [{ id: 'acknowledged', name: labelAcknowledged }]
     },
-    { name: 'statuses', value: [{ id: 'OK', name: labelOk }] },
+    {
```

Comment:
if criterias of type static , we don t add a formattedName , ref (./Resources/Filter/Criterias/searchQueryLanguage/index.ts)
line 120 , otherwise i should add the objectType to remove the ambiguity